libsignal-protocol/src/keys/private.rs

```diff
@@ -50,17 +50,23 @@ impl PrivateKey {
         }
     }
 
-    /// returns this private key as a base64 encoded string
-    pub fn as_base64(&self) -> Result<String, Error> {
+    /// Get an immutable reference to the underlying private key data.
```

Comment:
I don't think this returns a reference to the underlying data, doesn't this allocate and copy it into a new buffer?

Reply:
Also, if it's indeed an allocating operation, I think the Rusty way of naming this would be `to_bytes()`. Correct me if I'm wrong!